renderer/src/components/UpdateBanner.tsx

```diff
@@ -5,12 +5,13 @@ import { openReleaseNotes, restartToUpdate } from '../lib/station-config'
 const UpdateBanner = () => {
   const [isUpdateAvailable, setIsUpdateAvailable] = useState<boolean>(false)
 
-  useEffect(() => {
-    window.electron.getUpdaterStatus()
-      .then(({ updateAvailable }) => setIsUpdateAvailable(updateAvailable))
-  })
+  const reload = async () => {
+    const updateStatus = await window.electron.getUpdaterStatus()
+    setIsUpdateAvailable(updateStatus.updateAvailable)
+  }
 
   useEffect(() => {
+    reload()
```

Comment:
I am not a React expert, but their docs explicitly warns against defining async functions outside of `useEffect`.

Quoting from https://reactjs.org/docs/hooks-faq.html#is-it-safe-to-omit-functions-from-the-list-of-dependencies
> **Is it safe to omit functions from the list of dependencies?**
> Generally speaking, no.
> (...)
> It’s difficult to remember which props or state are used by functions outside of the effect. This is why usually you’ll want to declare functions needed by an effect inside of it. Then it’s easy to see what values from the component scope that effect depends on.

Reply:
You're probably right. I followed a pattern we're using somewhere else. Maybe I should refactor it as well in a follow-up PR!

Reply:
20c7fb2